### PR TITLE
chore: add documentation on iss

### PR DESCRIPTION
**Pull request by FrankEssenberger**
Added documentation on the ISS field so it is used in the right way hopefully.

### CODE REVIEW

**Comment by FrankEssenberger** [comment]
I will add a test

File: packages/core/src/connectivity/scp-cf/destination/destination-accessor-types.ts [hunk 1]
```diff
@@ -29,9 +29,12 @@ export interface DestinationAccessorOptions {
   userJwt?: string;
 
   /**
-   * @hidden
+   * This property is only considered in case no userJwt is provided.
+   * It is meant for situations where you do not have a token e.g. background processes.
+   * The value for iss is the issuer field of a JWT e.g. https://<your-subdomain>.localhost:8080/uaa/oauth/token'
+   *
+   * ATTENTION: If this property is used no validation of the provided subdomain value is done in contrast to the JWT case.
```
Comment (marikaner) [inline packages/core/src/connectivity/scp-cf/destination/destination-accessor-types.ts:36]: I found this sentence a bit slippery, not sure if this is an improvement:
```suggestion
   * ATTENTION: If this property is used, no validation of the provided subdomain value is done. This is differs from how the `userJwt` is handled.
```

**Review by marikaner — APPROVED** [review]
Now that I see it, theoretically it can never happen, that userJwt and iss are set at the same time - this sounds as if we actually needed a conditional type here. I guess that would produce a breaking change as you already mentioned, but do you think for v2 that could be a relevant (and simple) improvement? If so maybe add a comment and/or a ticket.

**Comment by FrankEssenberger** [comment]
> Now that I see it, theoretically it can never happen, that userJwt and iss are set at the same time - this sounds as if we actually needed a conditional type here. I guess that would produce a breaking change as you already mentioned, but do you think for v2 that could be a relevant (and simple) improvement? If so maybe add a comment and/or a ticket.

Yes you are right. I created ticket for 2.0.

**Pull request merged**